src/circuit_builder.rs

```diff
@@ -307,23 +290,25 @@ mod tests {
     }
 
     #[test]
-    fn test_full() -> Result<()> {
+    fn test_full() {
```

Comment:
If you make the tests return `Result<(), dusk_plonk::error:Error>` you should be able to still use `?` in the tests. And I would really like to keep it on this way.

Reply:
I couldn't as some of the errors are from `std::fs::Error`